tower-balance/src/choose/p2c.rs

```diff
@@ -59,14 +59,24 @@ impl PowerOfTwoChoices {
 impl<K, L> Choose<K, L> for PowerOfTwoChoices
 where
     L: Load,
-    L::Metric: PartialOrd,
+    L::Metric: PartialOrd + ::std::fmt::Debug,
 {
     /// Chooses two distinct nodes at random and compares their load.
     ///
     /// Returns the index of the lesser-loaded node.
     fn choose(&mut self, replicas: Replicas<K, L>) -> usize {
         let (a, b) = self.random_pair(replicas.len());
-        if replicas[a].load() <= replicas[b].load() {
+
+        let a_load = replicas[a].load();
+        let b_load = replicas[b].load();
+        debug!(
```

Comment:
Is this info important enough to be in debug logs? My initial feeling is that it's noisy enough to be `trace!`, but you probably have better context on how often you need to see these.